### ISSUE TITLE
Typescript definitions?

### ISSUE THREAD

**Issue by dan-cooke** [functionalone/serverless-iam-roles-per-function#118]
Would it be possible to add TS definitions? 

I use TS for my serverless config, as I work in a monorepo and need to share configs across multiple projects.

**Comment by Victor-Learned** [comment]
FYI: this plugin is basically abandoned by the owner. Looks like it's been forked and is taking updates here https://github.com/andersquist/serverless-iam-roles-per-function

**Comment by ebisbe** [comment]
@Victor-Learned the project you link seems to be in a similar state. I mean there's not much to implement maybe...

**Comment by Victor-Learned** [comment]
@ebisbe Yeah at the time the owner of that repo pulled over a few fixes from ~6-9 months ago but perhaps lost interest. Probably worth reaching out since it looks like he at least fixed the pipeline issues as well.
